packages/config/config/devices/0x031e/lzw31-bsd_0.0-1.45.json

```diff
@@ -146,6 +146,78 @@
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
+		"12": {
+			"label": "Association Behavior",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 15,
+			"defaultValue": 15,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
```

Comment:
These options are best modeled as partial parameters. I think we have that exact same param in another LZW config

Reply:
I don't see that this parameter exists in the [manual](https://support.inovelli.com/portal/api/kbArticles/407409000001873356/attachments/1abnf924b37dde28f4ea1bba3d1a3911d7384/content?portalId=1ce1db55e4825f16146555b24f0f1753cbf5f7d83ca0a3d8b31013b2a2d69e1f&inline=true) or in the [firmware history](https://support.inovelli.com/portal/en/kb/articles/firmware-change-log-lzw31-dimmer-switch-black-series). Am I missing something?

Reply:
They dont have that one in the manual for some reason, but it is a valid parameter and you can find it in their OZW [config files](https://github.com/InovelliUSA/OpenZWave/blob/master/config/inovelli/lzw31-sn.xml) and their [SmartThings device handlers](https://github.com/InovelliUSA/OpenZWave/blob/master/config/inovelli/lzw31-sn.xml) right from Inovelli themselves.

Reply:
The SmartThings driver linked was provided by Inovelli to SmartThings. They appear to have undocumented parameters available.